src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+$primary-text-color: #060b35;
+$secondary-text-color: #616070;
+
+.card {
+  width: 198px;
+  height: 406px;
+  padding: 16px;
+  background-color: #fff;
+  box-sizing: border-box;
+  border-radius: 5px;
+
+  // border: 1px solid #ffffff;
+  transition: 300ms;
+
+  // &:hover {
+  //   transform: scale(120%);
+
+  //   & .card__title {
+  //     color: #34568b;
+  //   }
+  // }
```

Comment:
The hover effect for the card is commented out. If you intend to use this effect, consider uncommenting these lines to enable the hover transformation and title color change.